packages/snap-preact/components/src/components/Atoms/Button/Button.tsx

```diff
@@ -164,4 +164,4 @@ export interface ButtonLang {
 	button?: Lang<never>;
 }
 
-export type ButtonNames = 'close' | 'apply' | 'clear' | 'slideout' | 'filter-toggle';
+export type ButtonNames = 'close' | 'apply' | 'clear' | 'slideout' | 'sidebar-toggle' | 'sidebar-close' | 'close-filters' | 'apply-filters';
```

Comment:
Some of these should be removed, namely: 'sidebar-close', 'close-filters' and 'apply-filters'.